WordPress/Classes/Utility/Migration/ContentMigrationCoordinator.swift

```diff
@@ -58,30 +69,30 @@
     /// just let the user continue with the original intent in case of failure.
     ///
     /// - Parameter completion: Closure called after the export process completes.
-    func startAndDo(completion: ((Result<Void, ContentMigrationCoordinatorError>) -> Void)? = nil) {
+    func startAndDo(completion: ((Result<Void, MigrationError>) -> Void)? = nil) {
```

Comment:
👋 @dvdchr - it looks like Jetpack also runs this sequence and would write out an `.ineligible` failure at least once - do we want that? It [gets called from here](https://github.com/wordpress-mobile/WordPress-iOS/blob/24e403aba234a86ebee1e96343d8727b9155cc78/WordPress/Classes/System/WordPressAppDelegate.swift#L332).

I discovered this because I was testing `21.4` Jetpack with `21.3` WordPress and we're getting `true` for `isCompatibleWordPressAppPresent` since we only disabled the feature flags but not the URL schemes. 😬 

I'd appreciate another set of eyes on the problem, but I wanted to note two potential changes that will be semi-related to this PR:
- Adding a `if AppConfiguration.isWordPress` somewhere for the `startOnceIfNeeded` sequence so that it only runs in WordPress
- Removing the `wordpressmigration+v1://` as a `21.3` beta fix

Thanks!

Reply:
- WordPress check added in 42789939adc085145cd14416742b89845447bf21.
- Confirmed that `21.3` will be skipped. 👍